project.mml

```diff
@@ -87,15 +87,17 @@ Layer:
       table: |-
         (SELECT
             way, name, way_pixels, religion,
-            COALESCE(wetland, landuse, "natural") AS feature
+            COALESCE(wetland, landuse, military, "natural") AS feature
```

Comment:
I think we don't need military=danger_area in this layer. The layer `landuse-overlay` also works for lowzoom rendering.

We can even remove the landuse=military from this layer: the style for #landcover-low-zoom does not seem to use landuse=military.

Reply:
fixed